integration-tests/src/test/java/oracle/weblogic/kubernetes/ItKubernetesEvents.java

```diff
@@ -807,7 +807,7 @@ void testK8SEventsStartStopWatchingNSWithLabelSelector(boolean enableClusterRole
         .execute();
 
     logger.info("verify NamespaceWatchingStopped event is logged in namespace {0}", domainNamespace3);
-    checkNamespaceWatchingStoppedEvent(opNamespace, domainNamespace3, null, "Normal", timestamp,
+    checkNamespaceWatchingStoppedEvent(opNamespace, domainNamespace3, domainUid, "Normal", timestamp,
```

Comment:
domainUid should be null as before. NamespaceWatchingStopped event will not contin domainUID